ci/dhis2-verify-app.yml

```diff
@@ -83,31 +83,20 @@ jobs:
     #          with:
     #              node-version: 12.x
     #
-    #        - uses: c-hive/gha-yarn-cache@v1
-    #        - run: yarn install --frozen-lockfile
-    #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
     #        - name: End-to-End tests
     #          uses: cypress-io/github-action@v2
     #          with:
-    #              install: false
+    #              start: yarn start:nobrowser
+    #              wait-on: 'http://localhost:3000'
+    #              wait-on-timeout: 300
     #              record: true
     #              parallel: true
-    #              start: ${{ env.SERVER_START_CMD }}
-    #              wait-on: ${{ env.SERVER_URL }}
-    #              wait-on-timeout: 300
-    #              cache-key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-    #              group: 'e2e'
-    #              tag: ${{ github.event_name }}
     #          env:
-    #              CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
     #              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    #              COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
-    #              STORYBOOK_TESTING: true
-    #              SERVER_START_CMD: 'yarn cy:server'
-    #              SERVER_URL: 'http://localhost:5001'
+    #              CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+    #              CYPRESS_dhis2BaseUrl: https://debug.dhis2.org/dev
+    #              CYPRESS_dhis2ApiVersion: 37
+    #              CYPRESS_networkMode: stub
```

Comment:
Our approach has changed a little over time, so I've just updated it to what we use in the scheduler.

We could also decide to make this more generic. Or, we could document the things that need to be updated outside of this workflow, in the app. Say with an explanatory comment, or more in-depth in the readme.

Reply:
We could move the `BROWSER` variable to the `env` block, then you can just put `start: d2-app-scripts` as that should be the default.

Reply:
First of all, I'm not 100% sure anything would actually go wrong if we were to start the webpack server here without the `BROWSER=none`, but it seems to make sense to keep that in and also keep the comment that explains about it. Maybe just phrase it to describe what the code does, not what the dev should do:
> This serves the app without opening a browser

Second of all I was thinking about what command to actually put in there. I think when people are using this workflow, they're on a platform-app. When the platform app is initialised a `start` script is generated in the `package.json`, so I think we can also safely assume that executing `yarn start` would be an available command. So then couldn't we just put `start: BROWSER=none yarn start` on line 91?

Or alternatively use `BROWSER=none d2-app-scripts start` as you already suggest in the comment.

My main point is: I think we should be able to put a command in there that works out-of-the-box. IMO that would be preferred over requiring the user to add a script in the package.json first.

Reply:
Ah yeah that's nice

Reply:
> We could move the BROWSER variable to the env block, then you can just put start: d2-app-scripts as that should be the default.

Done 👍. I used a slightly different command: `yarn d2-app-scripts start`, as I wasn't sure whether the cypress action would automatically prefix it. I'll double check that.

Reply:
> My main point is: I think we should be able to put a command in there that works out-of-the-box. IMO that would be preferred over requiring the user to add a script in the package.json first.

Yeah that seems nice, I've gone with Viktor's suggestion of using d2-app-scripts directly. That should work regardless of the scripts in package.json.